<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Request</class>
 <widget class="QWidget" name="Request">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/Request/EstimateAndRequest.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Address should be like this :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DestinationTxt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>271</width>
     <height>79</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>&lt;Destination 0 Address&gt;  &lt;Destination 1 Address&gt; ... &lt;Final Destination Address&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;GalaxyName,PlanetName&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SourcTxt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>&lt;Source Address&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="VipBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VIP Trip</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>158</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Trip Cost : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="CostLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Trip Distance :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="DistanceLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
